### PR TITLE
avoid get integer return null

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

File: tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/XMLMemento.java [hunk 1]
```diff
@@ -203,7 +203,7 @@ public Float getFloat(String key) {
 	public Integer getInteger(String key) {
 		Attr attr = element.getAttributeNode(key);
 		if (attr == null)
-			return null;
+			return 0;
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/XMLMemento.java]: use -1 instead

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

File: tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/XMLMemento.java (hunk 1 above)
Comment (simonjhy) [reply tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/XMLMemento.java]: fix

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

**Comment by jtydhr88** [comment]
sent to Greg, thanks

**Pull request closed without merging**